iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverterTest.java

```diff
@@ -0,0 +1,168 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.NavigableMap;
+import java.util.TreeMap;
+
+import org.apache.hadoop.hbase.client.Result;
+import org.junit.Test;
+
+import com.google.common.primitives.UnsignedBytes;
+
+import eu.dnetlib.data.proto.FieldTypeProtos.Qualifier;
+import eu.dnetlib.data.proto.KindProtos.Kind;
+import eu.dnetlib.data.proto.OafProtos.Oaf;
+import eu.dnetlib.data.proto.OafProtos.OafRel;
+import eu.dnetlib.data.proto.ProjectOrganizationProtos.ProjectOrganization;
+import eu.dnetlib.data.proto.ProjectOrganizationProtos.ProjectOrganization.Participation;
+import eu.dnetlib.data.proto.RelMetadataProtos.RelMetadata;
+import eu.dnetlib.data.proto.RelTypeProtos.RelType;
+import eu.dnetlib.data.proto.RelTypeProtos.SubRelType;
+import eu.dnetlib.iis.importer.schemas.ProjectToOrganization;
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+* @author mhorst
+*/
+
+public class ProjectToOrganizationConverterTest {
+
+	private byte[] relationColumnFamilyBytes = "dummy".getBytes();
+	
+    private Oaf oaf = null;
+    
+    private Result result = mock(Result.class);
+    
+    private ResultApprover defaultApprover = new ResultApprover() {
+		@Override
+		public boolean approveBeforeBuilding(Oaf oaf) {
+			return true;
+		}
+	};
+    
+    
+    //------------------------ TESTS --------------------------
+    
+	@Test(expected = NullPointerException.class)
+    public void constructor_relationColumnFamilyBytes_NULL() throws Exception {
+    	new ProjectToOrganizationConverter(defaultApprover, null);
+    }
+	
+	@Test(expected = NullPointerException.class)
+    public void constructor_resultApprover_NULL() throws Exception {
+    	new ProjectToOrganizationConverter(null, relationColumnFamilyBytes);
+    }
+	
+    @Test(expected = NullPointerException.class)
+    public void buildObject_hbaseResult_NULL() throws Exception {
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			defaultApprover, relationColumnFamilyBytes);
+        converter.buildObject(null, oaf);
+    }
+    
+    @Test
+    public void buildObject_disapproved_candidates() throws Exception {
+        //given
+    	ProjectToOrganizationConverter converter = new ProjectToOrganizationConverter(
+    			new ResultApprover() {
```

Comment:
If you had a mock as a resultApprover you wouldn't have to override the approveBeforeBuilding method here.